app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ActiveRecord::Base
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  after_save :update_likes_counter
```

Comment:
- [ ] OPTIONAL
Good job for invoking `after_save` method on `update_likes_counter` method, however kindly mark `update_likes_counter` as private after the invoke, similar to :
```ruby
  # app/models/like.rb
  after_save :update_likes_counter

  private

  def update_likes_counter
    author.increment!(:likes_counter)
  end